scripting/get5_apistats.sp

```diff
@@ -196,13 +212,28 @@ static void CheckForLogo(const char[] logo) {
   }
 }
 
-static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode status,
+static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode statusCode,
                         int data) {
   if (failure || !successful) {
-    LogError("Logo request failed, status code = %d", status);
+    LogError("Logo request failed, status code = %d", statusCode);
```

Comment:
Status code is always 0 here, as it won't have received an HTTP response at all. Change it to something like "Network connection failed." or something.

Reply:
done